backend/compact-connect/stacks/transaction_monitoring_stack/transaction_history_processing_workflow.py

```diff
@@ -201,17 +201,20 @@ def __init__(
         )
 
         # Create alarm for failed step function executions
-        Alarm(
-            self,
-            f'{compact}-StateMachineExecutionFailedAlarm',
-            metric=state_machine.metric_failed(),
-            evaluation_periods=1,
-            threshold=1,
-            actions_enabled=True,
-            alarm_description=f'{state_machine.node.path} failed to collect transactions',
-            comparison_operator=ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
-            treat_missing_data=TreatMissingData.NOT_BREACHING,
-        ).add_alarm_action(SnsAction(persistent_stack.alarm_topic))
+        # TODO: we have been asked to disable this until all compacts have valid authorize.net # noqa: FIX002
+        #  accounts put into place to avoid unnecessary alerting. Once the system is ready to
+        #  go live, this alarm should be re-enabled.
+        # Alarm(
+        #     self,
+        #     f'{compact}-StateMachineExecutionFailedAlarm',
+        #     metric=state_machine.metric_failed(),
+        #     evaluation_periods=1,
+        #     threshold=1,
+        #     actions_enabled=True,
+        #     alarm_description=f'{state_machine.node.path} failed to collect transactions',
+        #     comparison_operator=ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+        #     treat_missing_data=TreatMissingData.NOT_BREACHING,
+        # ).add_alarm_action(SnsAction(persistent_stack.alarm_topic))
```

Comment:
Can we leave the alarm and just remove the action? I'd like to still see the CloudWatch alarm if we go looking for it.